### ISSUE TITLE
Idea: Represent edges between buildings with a pyramid

### ISSUE THREAD

**Issue by Richargh** [MaibornWolff/codecharta#316]
CC can currently show edges between buildings, drawn as arrows. I wonder if it makes sense to visualize it as a pyramid instead. 

The selected building is of full height. The buildings that direct edges go to are of a height slightly below that. The buildings that are reached transitively one height below that etc.

No clue if that actually helps us. Maybe it's better to just display a big number 1, 2, 3 etc. to show the edge hierarchy level. 

Long story short, I still want to find a great visualization for edges in our 3D tree map but I am still stuck.

**Comment by Richargh** [comment]
This went nowhere :)
